ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml

```diff
@@ -84,18 +103,22 @@ chain:
       *) echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"; exit 1;;
       esac
 
+      set -e
```

Comment:
doesn't this `set -e` need to be placed right after the csi test results?

Reply:
I figured in case other tests set +e it might make sense to do right before we run the actual tests instead of right after the results, shouldn't matter much